weave/trace_server/migrations/005_add_cost.up.sql

```diff
@@ -1,4 +1,8 @@
 CREATE TABLE llm_token_prices (
+     /*
+    `id`: The unique identifier for the call. This is typically a UUID.
+    */
+    id String,
```

Comment:
just flagging - very bad to add changes to a prior migration - i know we just landed this, but better form is to alter the table in yet another migration. Else, any commits between the land of the first table and now are invalid to deploy